rules/do-you-know-how-to-use-single-codebase-for-multiple-domains-with-tinacm-and-nextjs/rule.md

```diff
@@ -0,0 +1,163 @@
+---
+type: rule
+tips: ""
+title: Do you know how to use Single Codebase for Multiple Domains with TinaCMS
+  and Next.js
+seoDescription: Optimize your Next.js and TinaCMS project with a single codebase
+  to efficiently manage multiple domains, locations, and content.
+uri: do-you-know-how-to-use-single-codebase-for-multiple-domains-with-tinacm-and-nextjs
```

Comment:
Hi @PothieuG 

We want to avoid really long URLs https://www.ssw.com.au/rules/create-friendly-short-urls/

- [ ] Please reduce the words. E.g.single-codebase-for-multiple-domains-with-tinacms
- [ ] Remember to update the rule folder as well (must be the same)